keras_cv/models/generative/stable_diffusion/__internal__/layers/group_normalization.py

```diff
@@ -0,0 +1 @@
+from tensorflow_addons.layers import GroupNormalization
```

Comment:
We need to remove this and duplicate/porting/refactor it in keras-cv or keras repo (https://github.com/keras-team/keras-cv/issues/74)

Reply:
@ianstenbit is working on this for now, it’ll live in core Keras - temporarily as a CV internal API